lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -5,19 +5,19 @@ module LoginProtection
     class ShopifyDomainNotFound < StandardError; end
 
     included do
+      after_action :set_test_cookie if ShopifyApp.configuration.embedded_app?
```

Comment:
I don't see any tests added for these changes (just some slight modifications). Can you please add some?